pyproject.toml

```diff
@@ -5,6 +5,9 @@ requires = [
   "setuptools>=61.2",
   "setuptools_scm[toml]>=6.2",
   "wheel",
+  "extension-helpers",
+  "oldest-supported-numpy",
+  "cython"
```

Comment:
```suggestion
```

We should update the template to drop `oldest-supported-numpy` (https://github.com/OpenAstronomy/packaging-guide/issues/46) and `extension-helpers` is a duplicate here, and we don't use `cython`. I wonder if we should remove cython from the template or not :thinking:  we use it for the example stuff, but you don't always *need* it if compiled extensions is true.